plugins/Toolbox/src/Toolbox.py

```diff
@@ -355,7 +379,7 @@ def canDowngrade(self, package_id: str) -> bool:
 
     @pyqtSlot(str, result = bool)
     def isInstalled(self, package_id: str) -> bool:
-        return self._package_manager.isPackageInstalled(package_id)
+        return self._package_manager.isPackageInstalled(package_id) or self._isOldPlugin(package_id)
```

Comment:
Even plugins which are not installed will now appear as such with an update button. Not ideal and kind of misleading.